samples/activity-update/README.md

```diff
@@ -0,0 +1,116 @@
+# Bot Framework activity update sample.
+
+This Microsoft Teams bot uses the [msbotbuilder-go](https://github.com/infracloudio/msbotbuilder-go) library. It shows how to create a simple bot that accepts input from the user and echoes response with an attachment.
+
+## Run the example
+
+#### Step 1: Register MS BOT
+
+Follow the official [documentation](https://docs.microsoft.com/en-us/microsoftteams/platform/bots/how-to/create-a-bot-for-teams#register-your-web-service-with-the-bot-framework) to create and register your BOT.
+
+Copy `APP_ID` and `APP_PASSWORD` generated for your BOT.
+
+Do not set any messaging endpoint for now.
+
+#### Step 2: Run echo local server
+
+Set two variables for the session as `APP_ID` and `APP_PASSWORD` to the values of your BotFramework `APP_ID` and `APP_PASSWORD`. Then run the `main.go` file.
+
+```bash
+export APP_PASSWORD=MICROSOFT_APP_PASSWORD
+export APP_ID=MICROSOFT_APP_ID
+
+go run main.go
+```
+
+This will start a server which will listen on port `3978`
+
+#### Step 3: Expose local server with ngrok
+
+Now, in separate terminal, run [ngrok](https://ngrok.com/download) command to expose your local server to outside world.
+
+```sh
+$ ngrok http 3978
+```
+
+Copy `https` endpoint, go to [Bot Framework](https://dev.botframework.com/bots) dashboard and set messaging endpoint under Settings.
+
+#### Step 4: Test the BOT
+
+You can either test BOT on BotFramework portal or you can create app manifest and install the App on Teams as mentioned [here](https://docs.microsoft.com/en-us/microsoftteams/platform/bots/how-to/create-a-bot-for-teams#create-your-app-manifest-and-package).
+
+
+## Understanding the example
+
+The program starts by creating a handler struct of type `activity.HandlerFuncs`.
+
+This struct contains definition for the `OnMessageFunc` field which is a treated as a callback by the library on the respective event.
+In this sample when GetCard command is issued from chat application adaptive card with input box is sent from this bot. Once submit button on card is clicked the card is deleted.
```

Comment:
I believe the application also update the text on the card - "Changed Activity" which demonstrates the Update activity handling. Please document this behavior.

Reply:
@sbawaskar please let me know once ready

Reply:
Sure

Reply:
Done thanks!

Reply:
@PrasadG193 , updated doc. Please check.